### PR TITLE
Feature/753 add distance filter

### LINKED ISSUE #752: Add distance filter to search page
It's enough to add the filter without the on/off toggle in this issue. The toggle will be added later.

tilan-listaus-etaisyys_sidebyside

### LINKED ISSUE #753: Add on/off toggle to search page distance filter
tilan-listaus-etaisyys_sidebyside

### PR DESCRIPTION
**Pull request by YoYuUm**
Closes #753. Closes #752

I prioritized this issues because I though it was going to make it easier to implement #623.

### CODE REVIEW

File: app/pages/search/controls/PositionControl.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import classnames from 'classnames';
+import React, { PropTypes } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+// import Modal from 'react-bootstrap/lib/Modal';
```
Comment (lraivio) [inline app/pages/search/controls/PositionControl.js]: - [ ] Remove code that is commented out.

File: app/pages/search/controls/PositionControl.spec.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+describe('pages/search/controls/PositionControl', () => {
+  it('renders a div.app-PositionControl', () => {
+    const wrapper = getWrapper();
+    expect(wrapper.is('div.app-PositionControl')).to.be.true;
+  });
+
+  describe('Button', () => {
+    it('does not have active class if not geolocated', () => {
+      const button = getWrapper().find(Button);
```
Comment (lraivio) [inline app/pages/search/controls/PositionControl.spec.js]: - [ ] Here it's better to explicitly have `getWrapper({ geolocated: false })...` so the test isn't relying on defaults in the getWrapper and break if the value there would for some reason change. Plus the reader can immediately see just from looking at this one test what props affect the active class.

File: app/pages/search/controls/SearchControlsContainer.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// import isEqual from 'lodash/isEqual';
```
Comment (lraivio) [inline app/pages/search/controls/SearchControlsContainer.js]: - [ ] Remove this.

File: app/pages/search/controls/_search-controls.scss [hunk 4]
```diff
@@ -100,6 +100,28 @@
...
+  &__show-button {
+    @include search-control-button($medium-gray);
+    font-size: 1.5em;
+    &.app-PositionControl__active {
+      @include search-control-button($copper);
+      font-size: 1.5em;
+    }
+  }
+
+  // &__modal {
```
Comment (lraivio) [inline app/pages/search/controls/_search-controls.scss]: - [ ] You can remove this as well.

**Comment by YoYuUm** [comment]
fixed

**Pull request merged**